src/commands/nsec3hash.rs

```diff
@@ -4,12 +4,16 @@ use domain::base::iana::nsec3::Nsec3HashAlg;
 use domain::base::name::Name;
 use domain::base::ToName;
 use domain::rdata::nsec3::{Nsec3Salt, OwnerHash};
+use lexopt::Arg;
 // use domain::validator::nsec::nsec3_hash;
 use octseq::OctetsBuilder;
 use ring::digest;
 use std::str::FromStr;
 
+use super::LdnsCommand;
+
 #[derive(Clone, Debug, clap::Args)]
+#[command(args_override_self = true)]
```

Comment:
I believe that this should be default behaviour for all CLI tools so that it's more convenient to create aliases and override the behaviour of an alias. Not really an issue for this tool, but I think we should just enable it on all tools.

Reply:
I'm not sure to what you are referring regarding aliases. Isn't it confusing to not error out when an arg that is expected once is given more than once?

Reply:
As an example of overriding aliases, with `ls`, you can do this:
```bash
alias my_cool_ls="ls --color --hyperlink --human-readable --classify"
```
but then if you want all those options but not the color bit, you can do
```
my_cool_ls --color=none
```
While keeping all the other fancy options.

Reply:
I'm not particularly swayed either way.  I like the convenience of the aliases but I don't think the ldns commands would be used that way and I agree that it can cause some confusion, particularly when hidden away in a script somewhere.